packages/backend/server/package.json

```diff
@@ -31,7 +31,6 @@
     "@nestjs/apollo": "^12.2.2",
     "@nestjs/common": "^10.4.15",
     "@nestjs/core": "^10.4.15",
-    "@nestjs/event-emitter": "^2.1.1",
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/hTwOityLamd4hitrae7M/5bba6644-761a-4edb-a44e-338b784f5ebf.png)

should refactor the payment plugin too.